<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPPPUser</class>
 <widget class="QDialog" name="DlgPPPUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Identificación usuario PPPoE</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="pppUserPassLineEdit">
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Usuario</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>IP Estática</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="pppUserNameLineEdit"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QFancyComboBox" name="pppProfileComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Perfil</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="pppUserPassCreateButton">
        <property name="text">
         <string>crear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="pppUserNameCopyButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../iconos.qrc">
          <normaloff>:/iconos/img/Copy-icon.png</normaloff>:/iconos/img/Copy-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="pppUserPassCopyButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../iconos.qrc">
          <normaloff>:/iconos/img/Copy-icon.png</normaloff>:/iconos/img/Copy-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Contraseña</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QFancyComboBox" name="staticIPComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Instalador</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Información instalación</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="6" column="2" rowspan="3">
       <widget class="QGroupBox" name="wifi5GroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>WiFi router/AP ac (5 Ghz)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>SSID</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="wifi5SSIDLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>WPA</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="wifi5WPALineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="ontSNComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="serviceTypeComboBox"/>
      </item>
      <item row="0" column="3" rowspan="10">
       <widget class="QGroupBox" name="lanGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Red Local</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="2">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>DMZ</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_19">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Puertos</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>IP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="3">
          <widget class="QLineEdit" name="lanDMZLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lanIPLineEdit"/>
         </item>
         <item row="2" column="0" colspan="6">
          <widget class="QPortForwardTableWidget" name="lanPortsTableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QToolButton" name="delPortButton">
           <property name="text">
            <string> - </string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QToolButton" name="addPortButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Número serie ONT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QGroupBox" name="wifi2GroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>WiFi router/AP bgn (2,4 Ghz)</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLineEdit" name="wifi2SSIDLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_22">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>WPA</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>SSID</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="wifi2WPALineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" rowspan="3" colspan="2">
       <widget class="QGroupBox" name="voipGroupBox">
        <property name="title">
         <string>VozIP</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Usuario</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="voipServerLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="voipPhoneNumber"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="voipUserPass"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Teléfono</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Contraseña</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Servidor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="voipUserName"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Notas Instalación</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Tipo servicio</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="iocCheckBox">
        <property name="text">
         <string>Internet por coaxial</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="tofCheckBox">
        <property name="text">
         <string>Tiene TV por la fibra</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QTextEdit" name="installNotesTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Identificación del cliente</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Notas cliente</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Email</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFancyComboBox" name="clientCityComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Población</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="clientPhonesLineEdit"/>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="clientAddressLineEdit"/>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QTextEdit" name="clientNotesTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="clientNameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Dirección</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLineEdit" name="clientEmailLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Nombre</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Teléfonos</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFancyComboBox" name="installerComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copyInfoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Copiar datos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyDataButton">
       <property name="text">
        <string>Crear usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clientLogsButton">
       <property name="text">
        <string>Registros</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="infoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFancyComboBox</class>
   <extends>QComboBox</extends>
   <header>Widgets/QFancyComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QPortForwardTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Widgets/QPortForwardTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pppUserNameLineEdit</tabstop>
  <tabstop>pppUserNameCopyButton</tabstop>
  <tabstop>pppUserPassLineEdit</tabstop>
  <tabstop>pppUserPassCreateButton</tabstop>
  <tabstop>pppUserPassCopyButton</tabstop>
  <tabstop>pppProfileComboBox</tabstop>
  <tabstop>staticIPComboBox</tabstop>
  <tabstop>clientNameLineEdit</tabstop>
  <tabstop>clientPhonesLineEdit</tabstop>
  <tabstop>clientAddressLineEdit</tabstop>
  <tabstop>clientCityComboBox</tabstop>
  <tabstop>clientEmailLineEdit</tabstop>
  <tabstop>clientNotesTextEdit</tabstop>
  <tabstop>voipGroupBox</tabstop>
  <tabstop>voipPhoneNumber</tabstop>
  <tabstop>voipUserName</tabstop>
  <tabstop>voipUserPass</tabstop>
  <tabstop>voipServerLineEdit</tabstop>
  <tabstop>wifi2GroupBox</tabstop>
  <tabstop>wifi2SSIDLineEdit</tabstop>
  <tabstop>wifi2WPALineEdit</tabstop>
  <tabstop>wifi5GroupBox</tabstop>
  <tabstop>wifi5SSIDLineEdit</tabstop>
  <tabstop>wifi5WPALineEdit</tabstop>
  <tabstop>lanGroupBox</tabstop>
  <tabstop>lanIPLineEdit</tabstop>
  <tabstop>lanDMZLineEdit</tabstop>
  <tabstop>addPortButton</tabstop>
  <tabstop>delPortButton</tabstop>
  <tabstop>lanPortsTableWidget</tabstop>
  <tabstop>installNotesTextEdit</tabstop>
  <tabstop>copyInfoButton</tabstop>
  <tabstop>applyDataButton</tabstop>
  <tabstop>clientLogsButton</tabstop>
 </tabstops>
 <resources>
  <include location="../iconos.qrc"/>
 </resources>
 <connections/>
</ui>
